test/tabs_test.js

```diff
@@ -93,6 +93,7 @@ describe('Tabs', function() {
 
   describe('selecting new tab (2)', function() {
     before(function(done) {
+      const browser = new Browser();
```

Comment:
Closing a tab is not supposed to remove any event listeners, but also closing the tab also should not trigger a `focus` event on that tab. We only trigger that event when opening or changing focus. So that would be a bug in the code.